docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -164,6 +164,16 @@ You should see something like the following:
 hot-restart-persistence-hazelcast-0 hot-restart-persistence-hazelcast-1 hot-restart-persistence-hazelcast-2
 ----
 
+=== Data Recovery Timeout
+
+To choose a data recovery timeout, you can use `dataRecoveryTimeout`.
+The field takes an integer value representing the timeout in seconds.
```

Comment:
To choose a data recovery timeout, you can use `dataRecoveryTimeout`. The field takes an integer value representing the timeout in seconds and uses this value to set [validation-timeout-seconds, data-load-timeout-seconds](https://docs.hazelcast.com/hazelcast/5.1/storage/configuring-persistence#global-persistence-options) Hazelcast Persistence options.

Reply:
done --> https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/31/files